sheeprl/configs/algo/ppo_torchrl.yaml

```diff
@@ -0,0 +1,53 @@
+defaults:
```

Comment:
What config is this?

Reply:
Something that has slipped through the PR: I've deleted it